packages/plugin-typescript/src/lib/schema.unit.test.ts

```diff
@@ -0,0 +1,69 @@
+import { describe, expect, it } from 'vitest';
+import {
+  type TypescriptPluginOptions,
+  typescriptPluginConfigSchema,
+} from './schema.js';
+
+describe('typescriptPluginConfigSchema', () => {
+  const tsConfigPath = 'tsconfig.json';
+
+  it('accepts a empty configuration', () => {
+    expect(() => typescriptPluginConfigSchema.parse({})).not.toThrow();
+  });
+
+  it('accepts a configuration with tsConfigPath set', () => {
+    expect(() =>
+      typescriptPluginConfigSchema.parse({
+        tsConfigPath,
+      } satisfies TypescriptPluginOptions),
+    ).not.toThrow();
+  });
+
+  it('accepts a configuration with tsConfigPath and empty onlyAudits', () => {
+    expect(() =>
+      typescriptPluginConfigSchema.parse({
+        tsConfigPath,
+        onlyAudits: [],
+      } satisfies TypescriptPluginOptions),
+    ).not.toThrow();
+  });
```

Comment:
These tests are out of sync with the schema - expects `tsconfig`, not `tsConfigPath`.